spec/facebook_client_spec.rb

```diff
@@ -78,6 +84,21 @@
       response.data["id"].nil?.should be false
     end
 
+    it 'can get locations' do
+      token_response = config.client.page_access_token(config.settings["page_id"])
+      access_token = token_response.data["access_token"]
+      location_fields = 'id, name, name_with_location_descriptor, username, picture, access_token, category, link, location, parent_page'
+      response = config.client.locations(fields: location_fields, limit: 1000, access_token: access_token)
+      response.successful?.should be true
+    end
+
+    # TODO: write ticket to work on figuring out how to test it on voyager
+    xit 'can get company info' do

```

Comment:
this should work as long as the page_id is for a page that is administered by the give access token

`it 'can get company info' do`
      `  response = config.client.company_info(config.settings["page_id"], fields: 'id, name, name_with_location_descriptor, username, picture, access_token, category, link, location')`
     `  response.successful?.should be true`
      `  response.data["id"].should eq(config.settings["page_id"].to_s)`
    `end`